lib/deps.js

```diff
@@ -54,8 +54,18 @@ const loadModule = (name) => {
   const subKeys = Object.keys(pkg.exports).map((key) => key.substring(2));
   const subNames = subKeys.filter(validSubmodules);
   for (const subName of subNames) {
-    const sub = appRequire(name + '/' + subName);
-    lib[subName] = sub;
+    try {
+      const sub = appRequire(name + '/' + subName);
+      if (lib[subName] && lib[subName] === sub[subName]) continue;
```

Comment:
Duplicate expression: `lib[subName]`

Reply:
This check is actually necessary here because in every other case (except the one I described above) `lib[subName]` and `sub[subName]` would be equal to `undefined`.